### ISSUE TITLE
Can't locate object method "_send_request" via package "Net::Amazon::S3::Client"

### ISSUE THREAD

**Issue by Mortivor** [rustyconover/net-amazon-s3#112]
Debian 11, Version of Net::Amazon::S3::Client is 0.97.
```
$ dpkg --list | grep amazon
ii  libnet-amazon-s3-perl                       0.97-1                          all          Amazon S3 Perl API - Simple Storage Service
```
I want to get the metadata of an existing object (checked with a query via Postman). Code is as follows:
```perl
my $bucket = $s3->bucket(name => BUCKET_NAME);
my $object = $bucket->object(key => OBJ_KEY);
my %metadata = %{$object->head};
warn Dumper(\%metadata);
```
On execution this results in
```
Can't locate object method "_send_request" via package "Net::Amazon::S3::Client" at /usr/share/perl5/Net/Amazon/S3/Client/Object.pm line 402.
```

**Comment by Mortivor** [comment]
Same with 0.98

**Comment by fkoyer** [comment]
I had this error as well. Looks like there is no `_send_request` method. I created PR #114 to fix the issue.

**Comment by happy-barney** [comment]
Solved in #116 / release v0.99 - also added missing test
